include/deal.II/multigrid/multigrid.h

```diff
@@ -99,9 +99,8 @@ class Multigrid : public Subscriptor
             Cycle                              cycle = v_cycle);
 
   /**
-   * Experimental constructor for cases in which no DoFHandler is available.
-   *
-   * @warning Not intended for general use.
+   * Constructor. Same as above but you can determine the multigrid
+   * levels to use yourself.
```

Comment:
One problem I have with this method is that we don't have the ``DoFHandler`` here so we can not check if the maxlevel the user specifies is valid.